cluster/aws/util.sh

```diff
@@ -596,9 +601,29 @@ function create-dhcp-option-set () {
   $AWS_CMD associate-dhcp-options --dhcp-options-id ${DHCP_OPTION_SET_ID} --vpc-id ${VPC_ID} > $LOG
 }
 
+function set-preset-variables-if-have {
+  if [[ "${PRESET_INSTANCES_ENABLED:-}" != "yes" ]]; then
+    return
+  fi
+  KUBE_MASTER_IP=$PRESET_KUBE_MASTER_IP
+  MASTER_INTERNAL_IP=$PRESET_MASTER_INTERNAL_IP
+  KUBE_MINION1_IP=$PRESET_KUBE_MINION1_IP    
+  if [[ -n ${PRESET_KUBE_MINION2_IP:-} ]]; then
+    KUBE_MINION2_IP=$PRESET_KUBE_MINION2_IP
+    NUM_NODES=2
+  else
+    NUM_NODES=1
+  fi
+  if [[ -z ${JENKINS_HOME:-} ]]; then    
+    ACCESS_FILE=${HOME}/ArktosE2E
```

Comment:
This looks like a security concern

Reply:
This has been discussed. The preset machines have known security issue.